frontend/src/modules/search/components/SearchBox.js

```diff
@@ -450,6 +450,7 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
                 onBlur={ this.unsetFocus }
                 onChange={ this.updateSearchInput }
                 onFocus={ this.setFocus }
+                onKeyDown={ this.performSearch }
```

Comment:
The reason for my previous is that here, on reading this line, I think that you perform the search on each key stroke.